main.nf

```diff
@@ -488,25 +1025,100 @@ def bqsr_version(task) {
 	"""
 }
 
-//Collect various QC data: 
+// When rerunning sample
+process dnascope {
+	cpus 54
+	memory '100 GB'
+	// 12 GB peak giab //
+	time '4h'
+	tag "$id"
+	container  "${params.container_sentieon}"
+
+	input:
+		tuple val(group), val(id), val(bam), val(bai), val(bqsr)
+
+	output:
+		tuple val(group), val(id), path("${id}.dnascope.gvcf.gz"), path("${id}.dnascope.gvcf.gz.tbi"), emit: gvcf_tbi
+		path "*versions.yml", emit: versions
+
+	when:
+		params.varcall
+
+	// TODO: Move shards out to some config file:
```

Comment:
Yes. Looks a bit risky right now if we would change reference genome, if they have slightly longer chromosomes